SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md

```diff
@@ -5,7 +5,7 @@ The CBL-Mariner SPEC files originated from a variety of sources with varying lic
 | CentOS | [MIT](https://www.centos.org/legal/#licensing-policy) | crash-ptdump-command <br> delve <br> fstrm <br> nodejs-nodemon <br> rhnlib <br> rt-setup <br> rt-tests <br> rtctl <br> tuned |
 | Ceph source | [LGPL2.1](https://github.com/ceph/ceph/blob/master/COPYING-LGPL2.1) | ceph |
 | Debian | [MIT](https://opensource.org/licenses/MIT) | prometheus-process-exporter |
```

Comment:
We might want to rebase because of:
- #7505
- #7529

Reply:
merged.